packages/components/src/components/languageSupport/lezerDebug.ts

```diff
@@ -0,0 +1,105 @@
+import { Input, NodeType, Tree, TreeCursor } from "@lezer/common";
+
+enum Color {
+  Red = 31,
+  Green = 32,
+  Yellow = 33,
+}
+
+function colorize(value: any, color: number): string {
+  return String(value);
+  // return "\u001b[" + color + "m" + String(value) + "\u001b[39m"
+}
+
+function focusedNode(cursor: TreeCursor): {
+  readonly type: NodeType;
+  readonly from: number;
+  readonly to: number;
+} {
+  const { type, from, to } = cursor;
+  return { type, from, to };
+}
+
+export function printTree(
```

Comment:
Sorry, I've just realised that this module is a debug module.

I'd be leaning mildly towards removing this module if it isn't used. If you want, an option could possibly be set that allows you to use this debug module maybe? And uncomment the code above that calls it?

Reply:
Oh, I guess I need to read through all of review before starting commenting then :D

It would be nice to save it somehow for further testing/development, it does make hacking on the parser much more fun